tftrt/examples/image-classification/image_classification.py

```diff
@@ -83,6 +95,15 @@ def model_fn(features, labels, mode):
     # preprocess function for input data
     preprocess_fn = get_preprocess_fn(model)
 
+    def load_and_preprocess_image(path):
```

Comment:
Can we combine `load_and_preprocess_image` and `get_preprocess_fn`?
It would need a mode argument.